core/test/integration/model/model_posts_spec.js

```diff
@@ -475,12 +442,11 @@ describe('Post Model', function () {
                 }).catch(function (err) {
                     should.exist(err);
                     (err instanceof errors.ValidationError).should.eql(true);
-                    done();
                 });
             });
 
-            it('draft -> scheduled: invalid published_at update', function (done) {
-                PostModel.findOne({status: 'draft'}).then(function (results) {
+            it('draft -> scheduled: invalid published_at update', function () {
+                return PostModel.findOne({status: 'draft'}).then(function (results) {
```

Comment:
This is one of the few cases in which we need to use `done`.

The reason behind this is that we are checking to make sure that an error _is_ thrown. In this case, I believe that there should be another `then` at the bottom, making it look something like this:

```js
.then(function () {
    done(new Error('Error should have been thrown')); // we use `done` with an error because we aren't supposed to get here
}).catch(function (err) {
    // all the checks there should be present here as well
    done(); // this is called to say the test is done
});
```